schedule/yast/maintenance/yast-mru-install-minimal-with-addons_15sp3@s390x.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: yast-mru-install-minimal-with-addons
+vars:
+  PATTERNS: base,minimal
+  YUI_REST_API: 1
+schedule:
+  access_beta: []
+  extension_module_selection:
+    - installation/module_registration/register_extensions_and_modules
+  add_on_product:
+    - installation/add_on_product/add_maintenance_repos
+    - installation/addon_products_sle
+  system_role: []
+  software:
+    - installation/select_patterns
```

Comment:
Please change to use 'installation/select_only_visible_patterns_from_top'

Reply:
changed.